packages/react-components/src/AccessibilityScenarios/Input.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Input } from '@fluentui/react-components/unstable';
```

Comment:
question: Is there any reason that Input is imported from `react-components/unstable` instead of `react-input`?

IMO Input should be imported from '@fluentui/react-input' instead of react-components, even though it is unstable it is a published component.

This might also fix the issue with the failing CI.

Reply:
Thanks, it works and also fixed the failing check.